### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by filiplazovic**
WIP

### CODE REVIEW

**Comment by filiplazovic** [comment]
@drewstone What else can we include here? It includes mostly info about the mixer. Should we include other circuits?

**Comment by drewstone** [comment]
I think some notes about the anchors and vanchor is good too.

File: README.md [hunk 1]
```diff
@@ -13,39 +13,44 @@ Many thanks to the following people for help and insights in both learning and i
...
+For the circuits implemented in this repo, we have setups in the [setup](https://github.com/webb-tools/arkworks-gadgets/tree/master/arkworks-circuits/src/setup) directory. This folder contains circuit-specific setup helpers for creating your provers and verifiers for your circuits from the previous section.
 
-The circuit-specific files of the setup section contain tests and circuit definitions that instantiate circuits w/ different configurations of hash gadgets and merkle tree gadgets. This is the primary place where one fixes the exact instantiations of a specific circuit.
+The circuit-specific files of the setup section contain tests and circuit definitions that instantiate circuits w/ different configurations of hash gadgets, merkle tree gadgets and elliptic curves. This is the primary place where one fixes the exact instantiations of a specific circuit.
 
 Each application-specific file in `src/setup` encapsulates the full-setup of a zero-knowledge gadget's prover and verifier. There are currently application-specific gadgets for:
```
Comment (drewstone) [inline README.md:49]: @filiplazovic this should be fixed to point to the right location.

**Review by drewstone — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -67,28 +76,91 @@ Any instantiation of a zero-knowledge mixer circuit requires that all data provi
...
+1. Using the input Utxos and corresopnding merkle paths, we are calculating the root hashes for each Utxo.
+2. We are checking if the root hash of each Utxo is a member of a root set. We are doing this with SetGadget.
+3. Using the output Utxos we are proving the leaf creation from passed private inputs.
+4. We are making sure that sum of input amounts plus the public amount is equal to the sum of output amounts.
+
+### Utxos
+
+// TODO: Better describe Utxo's
+
+Utxo is another name for a leaf. VAnchor allows for multiple Utxo's to be deposited or withdrawn in the same transaction.
```
Comment (filiplazovic) [inline README.md:148]: Still thinking how to explain this better

Comment (drewstone) [reply README.md:148]: UTXOs stand for unspent transaction outputs. Each UTXO represents a shielded balance that can be spent in the system. In order to create new UTXOs one must prove ownership over existing UTXOs that have at least as much balance as the newly created one. 

UTXOs contain a value, denoting the amount contained in the UTXO, the chain ID where the UTXO is meant to be spent on, and secret data relevant for creating zero-knowledge proofs of ownership and membership over.

UTXOs are deposited and stored in on-chain merkle trees first by serializing its components and then by hashing the serialized data before insertion. Each hash can be considered a commitment to a UTXO. In order to create new UTXOs from old ones, users must submit valid zero-knowledge proofs that satisfy constraints around the consistency of values and membership within a set of merkle roots.

Comment (drewstone) [reply README.md:148]: Let me know what you think @filiplazovic

Comment (filiplazovic) [reply README.md:148]: This is really good

**Pull request merged**